docusaurus.config.js

```diff
@@ -121,7 +125,7 @@ const config = {
       // },
       colorMode: {
         defaultMode: 'dark',
```

Comment:
```suggestion
        defaultMode: 'dark',
        disableSwitch: true,
        respectPrefersColorScheme: false,
```
Only dark mode would be enabled with this.